packages/dev-server/package.json

```diff
@@ -20,7 +20,11 @@
   "devDependencies": {
     "redux-logger": "^3.0.6",
     "webpack-dev-middleware": "^3.6.1",
-    "webpack-hot-middleware": "^2.22.0"
+    "webpack-hot-middleware": "^2.22.0",
+    "open": "^6.3.0"
```

Comment:
After going through the process of upgrading dependencies, I’m more wary of adding new ones, but it looks like this a very well maintained package, and the same one used by Storybook.